Makefile

```diff
@@ -113,7 +113,7 @@ build-and-push-images: ## Build and push images to local registry.
 		cd scripts && ./build-and-push-images.sh --image gradle-toolset
 		cd scripts && ./build-and-push-images.sh --image python-toolset
 		cd scripts && ./build-and-push-images.sh --image helm
-		cd scripts && ./build-and-push-images.sh --image node16-typescript-toolset --platform linux/amd64
+		cd scripts && ./build-and-push-images.sh --image node16-typescript-toolset
```

Comment:
I guess we can now undo https://github.com/opendevstack/ods-pipeline/commit/2e6726e10de61904e90a80ddefcc0b677d448904, right?

Reply:
done already in last commit, or?

Reply:
no, the whole target can simply be `cd scripts && ./build-and-push-images.sh`, which avoids hardcoding the images, so I'd prefer that :)

Reply:
also changed the bc of the related central tasks installation